_episodes/09-github.md

```diff
@@ -197,8 +197,109 @@ Already up-to-date.
 {: .output}
 
 Pulling has no effect in this case because the two repositories are already
-synchronized.  If someone else had pushed some changes to the repository on
-GitHub, though, this command would download them to our local repository.
+synchronized.  However, if there were changes in the GitHub remote repository
+that were not present in our current repository, this command would download
+them to our local repository.
+
+To demonstrate this, let's make a bare repository in a new directory.
```

Comment:
I'm a little nervous about using the word 'bare' since that is a specific term in git terminology.  Maybe empty instead?

Reply:
Good call. Changing...